src/main.rs

```diff
@@ -26,8 +26,8 @@ mod generics;
 mod tpf_fasta;
 use crate::tpf_fasta::tpf_fasta_mod::curate_fasta;
 
-mod exclude_seq;
-use crate::exclude_seq::exclude_seq_mod::filter_fasta;
+mod filter_fasta;
```

Comment:
Just wondering why we need to do this? Is this standard in Rust. We have had to do it in a few places when adding tests and not entirely sure why.